tags.go

```diff
@@ -28,32 +20,67 @@ func serializeTags(name string, tags map[string]string) string {
 	const prefix = ".__"
 	const sep = "="
 
-	if len(tags) == 0 {
+	// switch len(tags) {
+	switch len(tags) {
+	case 0:
 		return name
+	case 1:
+		for k, v := range tags {
+			return name + prefix + k + sep + replaceChars(v)
+		}
+		panic("unreachable")
+	case 2:
+		var a, b tagPair
+		for k, v := range tags {
```

Comment:
If you're going for speed, does unrolling this loop buy us anything?

Reply:
can't unroll map iteration since the keys are unknown.